### PR TITLE
[duplicati] Added Duplicati Chart

### PR DESCRIPTION
**Pull request by skaro13**
Signed-off-by: Simon Caron <simon.caron@protonmail.com>

#### Special notes for your reviewer:

Added a new chart to deploy LinuxServer's Duplicati image.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[radarr]`)

### CODE REVIEW

File: charts/duplicati/README.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+```console
+helm install --name my-release \
+  --set timezone="America/New York" \
+    billimek/duplicati
+```
+
+Alternatively, a YAML file that specifies the values for the above parameters can be provided while installing the chart. For example,
+
+```console
+helm install --name my-release -f values.yaml stable/duplicati
```
Comment (billimek) [inline charts/duplicati/README.md:99]: This should be `billimek/duplicati`

File: charts/duplicati/README.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+$ helm repo add billimek https://billimek.com/billimek-charts/
+$ helm install billimek/duplicati
+```
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+helm install --name my-release billimek/duplicati
```
Comment (billimek) [inline charts/duplicati/README.md:17]: With helm3 being the default now, we can probably drop the `--name` flag for the `helm install` command.

File: charts/duplicati/README.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+```
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```console
+helm delete my-release --purge
```
Comment (billimek) [inline charts/duplicati/README.md:25]: With helm3 being the default now, we can probably drop the `--purge` flag for the `helm delete` command.

**Comment by skaro13** [comment]
All done!

**Comment by billimek** [comment]
Thanks so much for contributing this chart, @skaro13!

**Pull request merged**